CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 #########
 
+Version 1.11.1
+--------------
+
+**New**:
+
+* pylint version can now be above 3.0.0 again
```

Comment:
I used pylint > 3 in pyproject.toml for easy testing, if we want compatibility to pylint 2.8.3+ this is going to be harder. But imo it's reasonable to use only 3.+ It supports python 3.7 to 3.13